SCPDiscordPlugin/Languages/zh-Hans.yml

```diff
@@ -0,0 +1,1419 @@
+  "Hcz096":"SCP-096 收容室",
+  "Hcz106":"SCP-106 收容室",
+  "Hcz939":"SCP-939 收容室",
+  "HczMicroHID":"Micro-HID 储存室",
+  "HczArmory":"重收容军械库",
+  "HczServers":"重收容机房",
+  "HczTesla":"特斯拉电网",
+  "EzCollapsedTunnel":"办公区被堵住的隧道",
+  "EzGateA":"办公区A大门",
+  "EzGateB":"办公区B大门",
+  "EzRedroom":"办公区被锁住的大门",
+  "EzEvacShelter":"办公区疏散避难所",
+  "EzIntercom":"办公区广播室",
+  "EzOfficeStoried":"办公区办公室",
+  "EzOfficeLarge":"办公区大型办公室",
+  "EzOfficeSmall":"办公区小型办公室",
+  "Outside":"外部区域",
+  "Pocket":"SCP-106口袋维度",
+  "HczTestroom":"重收容测试房间",
+
+  # 物品
+  #"None":"无",
+  "KeycardJanitor":"清洁工钥匙卡",
+  "KeycardScientist":"科学家钥匙卡",
+  "KeycardResearchCoordinator":"研究主管钥匙卡",
+  "KeycardZoneManager":"区域经理钥匙卡",
+  "KeycardGuard":"设施警卫钥匙卡",
+  "KeycardNTFOfficer":"九尾狐列兵钥匙卡",
+  "KeycardContainmentEngineer":"九尾狐收容工程师钥匙卡",
+  "KeycardNTFLieutenant":"九尾狐中士钥匙卡",
+  "KeycardNTFCommander":"九尾狐指挥官钥匙卡",
+  "KeycardFacilityManager":"设施主管钥匙卡",
+  "KeycardChaosInsurgency":"混沌分裂者破解装置",
+  "KeycardO5":"O5钥匙卡",
+  "Radio":"对讲机",
+  "GunCOM15":"COM-15",
+  "Medkit":"急救包",
+  "Flashlight":"手电筒",
+  "MicroHID":"Micro-HID",
+  "SCP500":"SCP-500",
+  "SCP207":"SCP-207",
+  "Ammo12gauge":".12号子弹",
+  "GunE11SR":"Epsilon-11标准步枪",
+  "GunCrossvec":"Crossvec冲锋枪",
+  "Ammo556x45":"5.56mm子弹",
+  "GunFSP9":"FSP9",
+  "GunLogicer":"轻机枪",
+  "GrenadeHE":"手榴弹",
+  "GrenadeFlash":"闪光弹",
+  "Ammo44cal":"44号子弹",
+  "Ammo762x39":"7.62mm子弹",
+  "Ammo9x19":"9mm子弹",
+  "GunCOM18":"COM-18",
+  "SCP018":"SCP-018",
+  "SCP268":"SCP-268",
+  "Adrenaline":"肾上腺素",
+  "Painkillers":"止痛药",
+  "Coin":"硬币",
+  "ArmorLight":"轻型护甲",
+  "ArmorCombat":"战术护甲",
+  "ArmorHeavy":"重型护甲",
+  "GunRevolver":"左轮手枪",
+  "GunAK":"AK步枪",
+  "GunShotgun":"散弹枪",
+  "SCP330":"SCP-330",
+  "SCP2176":"SCP-2176",
+  "SCP244a":"SCP-244-A",
+  "SCP244b":"SCP-244-B",
+  "SCP1853":"SCP-1853",
+  "ParticleDisruptor":"3-X离子分裂者",
+  "GunCom45":"COM-45",
+  "SCP1576":"SCP-1576",
+  "Coal":"煤块",
+  "Scp2536_2":"SCP-2536-2",
+  "Jailbird":"囚鸟",
+
+  # SCP-914 模式
+  "Rough":"超粗加工",
+  "Coarse":"粗加工",
+  "OneToOne":"1:1",
+  "VeryFine":"精加工",
+  "Fine":"超精加工",
+
+  # 钥匙卡权限
+  #"None":"无",
+  #"Checkpoints":"检查点权限",
+  "ExitGates":"出口大门权限",
+  #"Intercom":"广播室权限",
+  "AlphaWarhead":"Alpha核弹权限",
+  "ContainmentLevelOne":"收容1级权限",
+  "ContainmentLevelTwo":"收容2级权限",
+  "ContainmentLevelThree":"收容3级权限",
+  "ArmoryLevelOne":"武器1级权限",
+  "ArmoryLevelTwo":"武器2级权限",
+  "ArmoryLevelThree":"武器3级权限",
+  "ScpOverride":"SCP-override",
+
+  # 某些门的设置
+  "PortallessBreakableDoor":"可破坏门",
+  "BreakableDoor":"不可破坏门",
+  "door\\(Clone\\)":"克隆门"
+]
+
+# 对最终版本的消息进行处理，这是唯一影响玩家名称的正则表达式。
+final_regex: ["discord\\.gg":"","http://":"","https://":"","www\\.":"", "<@":"<", "<@!":"<!"]
+
+messages:
+  onroundstart:
+    message: "**回合已经开始**"
+    regex: []
+
+  onconnect:
+    # ipaddress - 字符串(string)
+    # steamid - 字符串(string)
+    # jointype - CentralAuthPreauthFlags
+    # region - 字符串(string)
+    message: "玩家 (Steam64位ID <var:steamid> IP <var.ipaddress>)正在尝试连接..."
```

Comment:
<var.ipaddress> should be <var:ipaddress>

But also, should this really add the ip address here? I think it would make more sense if all languages show the same information so they are just translations of the english one.

Reply:
Humm, I believe that displaying IP addresses is necessary in China,Because there are many cheaters in China, and their accounts cannot be completely banned
If you want to translate as is, I will modify it

Reply:
Ok, chinese servers can always change out that message in the override.yml file then.